Library/Formula/sgfutils.rb

```diff
@@ -0,0 +1,68 @@
+class Sgfutils < Formula
+  desc "collection of command-line utilities to work with SGF files"
+  homepage "http://homepages.cwi.nl/~aeb/go/sgfutils/html/sgfutils.html"
+  url "http://homepages.cwi.nl/~aeb/go/sgfutils/sgfutils-0.22.tgz"
+  sha256 "4ad83be91ac3a52dd8bd0766bfe403499430da96e680c5aea2f5381ba0b34923"
+
+  depends_on "openssl" => :build
```

Comment:
It's not _impossible_ for it to only be required at build-time, depending on what compile is doing with it, just very very very unusual.